vmm/src/coredump.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2022 ZTE Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::fs::File;
+
+#[derive(Clone)]
+pub struct MemCoredumpFileInfo {
+    pub offset: u64,
+    pub filesz: u64,
```

Comment:
Could you please rename this field in order to make it more comprehensive?

Reply:
Okay.